src/styles/ForecastBreakdown.css

```diff
@@ -0,0 +1,40 @@
+.forecast-breakdown {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  flex: 1 1 auto;
+  text-align: center;
+  min-height: 350px;
+  padding: 20px;
+}
+
+.forecast-breakdown__left {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.forecast-breakdown__left h3, .forecast-breakdown__left h3 + p {
```

Comment:
If you are targeting multiple classes they should be on separate lines to make it easier to read:
```
.forecast-breakdown__left h3,
.forecast-breakdown__left h3 + p {
```